config/routes/api/v2.rb

```diff
@@ -368,6 +368,7 @@
           get 'status/:type', :on => :member, :action => :get_status
           get :vm_compute_attributes, :on => :member
           get 'template/:kind', :on => :member, :action => :template
+          get :power_status, on: :member
```

Comment:
We have `put :power here`, don't we want to call the route `get :power` as well (for consistency)?

Reply:
I don't understand, the `put :power` already exists and we should be actually adding `get :power`. Please elaborate, why new action is being added for this.

Reply:
The new action is needed either way, otherwise you end up with duplicate methods in the controller. The only change possible here is the route name:

From
```
put :power, :on => :member
get :power_status, on: :member
```

To:
```
put :power, on: :member
get :power, on: :member, action: :power_status
```